packages/ActionMenu/src/ActionMenu.tsx

```diff
@@ -51,6 +51,8 @@ export interface ActionMenuProps extends React.ComponentProps<"div"> {
     renderReference: (
         props: React.HTMLProps<HTMLButtonElement>,
     ) => React.ReactElement;
+    /** Helper text to display below the options */
+    helperText?: string;
```

Comment:
I guess I'm just wondering if it is always just helper text here or should we allow the user to set whatever they want in the footer? Will this be a common thing?

Reply:
Checked in with a designer and indeed more versatile is better. Updated